src/components/menu/MobileMenu.vue

```diff
@@ -89,14 +93,25 @@ export default {
 
 		getNameByKey(e) {
 			return this.$store.getters['ui/getNameByKey'](e);
+		},
+
+		iconUrl(icon) {
+			switch (icon) {
+			case 'IconStatistics':
+				return this.IconStatistics;
+			case 'IconMosaics':
+				return this.IconMosaics;
```

Comment:
Why there is only `Statistic` and `Mosaic` icons?

Reply:
That 2 new icons from the design team.